### DISCUSSION TITLE
Individual display dimming and brightening by mouse possition

### DISCUSSION THREAD

**Discussion by marcoboerner** [MonitorControl/MonitorControl#910] [Ideas]
First of all your app is awesome and helped my productivity a lot!

One feature would be pretty great though, I have one large monitor which is my main desktop and I use my iMac screen as a secondary monitor. Most of the time I have the brightness set to its lowest on the iMac screen, only when I need to do something with it I increase the brightness, only to decrease it again once I'm done. This is already nicely done with MonitorControl and mouse position dependent brightness control.
However it would be amazing if this could be automated and basically introduce a screen dependent semi-sleep mode. So whenever moving the mouse pointer to the other screen it brightens it and sets its last known brightness setting. When the mouse pointer leaves the second screen it dims the display after a set period of time (maybe the same as in the display sleep settings)
I'm not sure how difficult it would be to implement, but considering that there is already a mouse pointer position dependent dimming, maybe it wouldn't even be that difficult?

**Comment by waydabber** [comment]
Hi @marcoboerner - it is possible to achieve to implement such a functionality, but I am not sure how widely used it would be. But I can imagine a scenario with a multi monitor setup that the display brightness focus could follow the mouse - it would be kinda cool. :) You can add a feature request regarding this under Issues and let's see what the dev team say about this one.
